gulpfile.mjs

```diff
@@ -109,7 +109,7 @@ gulp.task('watch-boot-script', () => {
       //
       // We could alternatively use `watchJS` to rebuild the bundle, but we'd
       // need to make its logging less noisy first.
-      'src/boot/**/*.js',
+      'src/boot/**/*.{js,ts,tsx}',
```

Comment:
I have added `tsx` everywhere where extensions had to be defined, just for consistency with other configs.

I think it's unlikely we need `tsx` in `boot`, but the impact is low, and if at some point we need it, errors are not so obvious and it's frustrating to see it doesn't work, until you realize what's missing.